ee/audit/interceptor.go

```diff
@@ -23,6 +23,8 @@ import (
 	"net/http"
 
 	"google.golang.org/grpc"
```

Comment:
the import order should be, standard library, dgraph imports followed by all the other external imports